Fixing prepare.md collection 2573 instructions (#32)
nimh-dsst/abcd-fasttrack2bids: docs/prepare.md

1. Click on `Data from Labs` on the left
1. Search for `abcd study`
1. Click on the checkbox to the left of `ABCD Study`, making sure that the `COLLECTION ID` is `2573`
1. Follow steps 6-11 from above. Step 9 may take a couple of hours for the `Filter Cart` to update.

> 1. Follow step 12, making sure to click the box that says `Include Associated Data Files` before creating the package

Reviewer: ```suggestion
1. When you follow step 12 from above, make sure to also click the box that says `Include Associated Data Files` before clicking `Create Data Package`
```
